var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -52,14 +53,18 @@ class Relion(CMakePackage, CudaPackage):
     depends_on('fltk', when='+gui')
     # cuda 9 not yet supported
     #  https://github.com/3dem/relion/issues/296
-    depends_on('cuda@8.0:8.99', when='+cuda')
+    # depends_on('cuda@8.0:8.99', when='+cuda')
+    depends_on('cuda@9:', when='version==beta-3')
     # use gcc < 5 when compiled with cuda 8
     conflicts('%gcc@5:', when='+cuda')
 
     def cmake_args(self):
         args = [
             '-DCMAKE_C_FLAGS=-g',
             '-DCMAKE_CXX_FLAGS=-g',
+            '-DALTCPU=ON',
+            '-DFORCE_OWN_TBB=ON',
+            '-DMKLFFT=ON',
```

Comment:
Just to make sure, these flags all apply to the older version as well?

Reply:
Good question...